src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx

```diff
@@ -111,7 +111,11 @@ export const ShowFeelingTemplate: React.FC<ShowFeelingTemplateProps> = ({
               >
                 <div className="feelingOfDay-name">
                   {feelingsEmojis[feelingsListObject[feelingId].category]}&nbsp;
-                  <span>{t(feelingsListObject[feelingId].content)}</span>
+                  <div>
```

Comment:
@Shubham-Praj please pull latest changes again to resolve the merge conflicts.

in this div create a class and put this css ```
{
display: -webkit-box;
-webkit-line-clamp: 2;
-webkit-box-orient: vertical;
overflow: hidden; }
```